examples/graph_data_demo.ipynb

```diff
@@ -24,31 +32,46 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
-   "metadata": {},
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
    "outputs": [],
    "source": [
     "import os\n",
     "import sys\n",
-    "import pandas as pd\n",
     "import pprint\n",
-    "sys.path.insert(0, '..')\n",
     "\n",
-    "import dataprofiler as dp\n",
+    "try:\n",
+    "    sys.path.insert(0, '..')\n",
+    "    import dataprofiler as dp\n",
+    "except ImportError:\n",
+    "    import dataprofiler as dp\n",
+    "\n",
     "data_path = \"../dataprofiler/tests/data\""
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
```

Comment:
what's the pycharm stuff?

Reply:
I removed it in vim, but when I committed it all got added back in. I think it is added in by the pre-commits